Conceptpower+Spring/src/main/java/edu/asu/conceptpower/servlet/web/ConceptAddController.java

```diff
@@ -50,111 +55,133 @@
  */
 @Controller
 public class ConceptAddController {
-	
-	private static final Logger logger = LoggerFactory
-			.getLogger(ConceptAddController.class);
-
-	@Autowired
-	private IConceptManager conceptManager;
-	
-	@Autowired
-	private IConceptListManager conceptListManager;
-
-	@Autowired
-	private IConceptTypeManger conceptTypesManager;
-
-	@Autowired
-	private ServiceRegistry serviceRegistry;
-	
-	@Autowired
-	private IIndexService indexService;
-	
-	@Value("#{messages['INDEXER_RUNNING']}")
+
+    private static final Logger logger = LoggerFactory.getLogger(ConceptAddController.class);
+
+    @Autowired
+    private IConceptManager conceptManager;
+
```

Comment:
@mkarthik90 this is probably due to line endings or tabs/spaces. What is used here? Spaces or tabs?